bigbluebutton-html5/imports/ui/components/media/container.jsx

```diff
@@ -80,8 +79,9 @@ class MediaContainer extends Component {
 
     const chromeErrorElement = (
       <div>
-        {intl.formatMessage(intlMessages.chromeExtensionError)}{' '}
-        <a href={CHROME_EXTENSION_LINK} target="_blank">
+        {intl.formatMessage(intlMessages.chromeExtensionError)}
+        {' '}
+        <a href={CHROME_EXTENSION_LINK} target="_blank" rel="noopener noreferrer">
```

Comment:
explanation for `  rel="noopener noreferrer" ` : https://stackoverflow.com/a/50709724

Reply:
fixing the lint error : Prevent usage of unsafe target='_blank' (react/jsx-no-target-blank)